### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Delf-Lee**
### 이런게 변경되었어요. 💍
- 쿼리 버그 수정
- 뉴스 내용 빈값 허용

### CODE REVIEW

**Comment by chul-hyun** [comment]
Hey @Delf-Lee,  
Your changes look good to me!

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;snack-news&#x2F;Snack-BE&#x2F;builds&#x2F;649410711">View build log</a>

###### TravisBuddy Request Identifier: e2862b80-4d8e-11ea-9ae4-55730306cb92

**Comment by NESOY** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- src/main/java/com/snack/news/repository/PicksRepositoryImpl.java  1
- src/main/java/com/snack/news/repository/NewsRepositoryImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4991072&bid=16476702)

File: src/main/java/com/snack/news/repository/NewsRepositoryImpl.java [hunk 1]
```diff
@@ -6,10 +6,7 @@
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 import javax.persistence.TypedQuery;
-import javax.persistence.criteria.CriteriaBuilder;
-import javax.persistence.criteria.CriteriaQuery;
-import javax.persistence.criteria.Predicate;
-import javax.persistence.criteria.Root;
+import javax.persistence.criteria.*;
```
Comment (NESOY) [inline src/main/java/com/snack/news/repository/NewsRepositoryImpl.java:9]: Codacy Issue found: [Avoid unused imports such as 'javax.persistence.criteria'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4991072)

File: src/main/java/com/snack/news/repository/PicksRepositoryImpl.java [hunk 2]
```diff
@@ -6,10 +6,7 @@
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 import javax.persistence.TypedQuery;
-import javax.persistence.criteria.CriteriaBuilder;
-import javax.persistence.criteria.CriteriaQuery;
-import javax.persistence.criteria.Predicate;
-import javax.persistence.criteria.Root;
+import javax.persistence.criteria.*;
```
Comment (NESOY) [inline src/main/java/com/snack/news/repository/PicksRepositoryImpl.java:9]: Codacy Issue found: [Avoid unused imports such as 'javax.persistence.criteria'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4991072)

**Pull request merged**